locales/en-US/strings.properties

```diff
@@ -145,6 +145,7 @@ pocket_read_more=Popular Topics:
 pocket_read_even_more=View More Stories
 pocket_more_reccommendations=More Recommendations
 pocket_learn_more=Learn More
```

Comment:
Is this being used anywhere else? If not, please remove it

Reply:
Removed